lib/IHP/Makefile.dist

```diff
@@ -134,7 +129,6 @@ clean: ## Resets all build and temporary files
 	rm -rf build/bin
 	rm -rf IHP/IHP/static/node_modules
 	rm -rf build/db
-	rm -f .envrc
```

Comment:
I'd still keep this line for a while. Otherwise when migrating from an older IHP version a clean rebuild using `make clean` will not fix all problems.